engine/src/io/DataLoader.cpp

```diff
@@ -94,7 +93,7 @@ std::unique_ptr<ral::frame::BlazingTable> data_loader::load_data(
 					}
 
 					if (schema.all_in_file()){
-						std::unique_ptr<ral::frame::BlazingTable> loaded_table = parser->parse(file_sets[file_set_index][file_in_set].fileHandle, user_readable_file_handles[file_index], fileSchema, column_indices_in_file);
+						std::unique_ptr<ral::frame::BlazingTable> loaded_table = parser->parse(file_sets[file_set_index][file_in_set].fileHandle, fileSchema, column_indices_in_file);
```

Comment:
why did you get rid of user_readable_file_handles? the point of this was to be able to let us tell the user on WHICH file they had an issue and make that file path readable to the user.

Reply:
user_readable_file_handles was not being used in any of the code except in one place to show an error. When we introduced the `data_handles` struct which contains the Uri, then user_readable_file_handles became useless and should have been taken out

Reply:
ok